### ISSUE TITLE
Ability to change background to white

### ISSUE THREAD

**Issue by ghosthoughts** [SublimeText-Markdown/MarkdownEditing#304]
Can I change the background to white in the theme? The combination of greyish background and light black text in the dark theme has been a strain on the eyes. I would prefer black text on white background, but with all other goodies of the theme.

**Comment by maliayas** [comment]
Yes you can, probably by modifying this line: https://github.com/SublimeText-Markdown/MarkdownEditing/blob/master/MarkdownEditor.tmTheme#L13

That color occurs several times in the file. Replace those as well.

**Comment by peterHeuz** [comment]
I would like to change the background to black. How can I find or integrate this file into my markdown installation (v2.1.6)?

**Comment by felixhao28** [comment]
You need to use follow manual installation instructions (with git). And in your Packages\MarkdownEditing you will see this file.

**Comment by peterHeuz** [comment]
Thanks for the reply.
So I copied the repository to my packages folder to the folder User. Since I use Sublime 3 my folder structure looks like this: %APPDATA/Roaming/Sublime Text 3/Packages/User/MarkdownEditing/.gitignore.
I restarted Sublime twice but don't see it listed as a package under the package control. I have some syntax highlighting for <code></code> though, but do not know if it comes from Sublime or MarkdownEditing.
